src/compose.js

```diff
@@ -20,7 +20,5 @@ export default function compose(...funcs) {
     return funcs[0]
   }
 
-  const last = funcs[funcs.length - 1]
-  const rest = funcs.slice(0, -1)
-  return (...args) => rest.reduceRight((composed, f) => f(composed), last(...args))
```

Comment:
the last part `last(...args)` in `reduceRight` is evalutating, not composing. That is broken